test/__snapshots__/bindings.test.ts.snap

```diff
@@ -79,15 +196,89 @@ class Name:
         )
 
 
+@jsii.data_type(
+    jsii_type=\\"1bc04b5291c26a46d918139138b992d2de976d6851d0893b0476b85bfbdfc6e6.NameObjectWithPropertiesAndBooleanAdditionalProperties\\",
+    jsii_struct_bases=[],
+    name_mapping={\\"name\\": \\"name\\"},
+)
+class NameObjectWithPropertiesAndBooleanAdditionalProperties:
+    def __init__(self, *, name: typing.Optional[builtins.str] = None) -> None:
```

Comment:
@eladb , I would expect here to contain `additional_properties: typing.Optional[typing.Mapping[builtins.str, builtins.any]]` or use any additional parameter to save in the private values.